packages/cms-lib/fileMapper.js

```diff
@@ -421,7 +429,11 @@ async function fetchFolderFromApi(input) {
     const node = isHubspot
       ? await downloadDefault(portalId, srcPath, { qs })
       : await download(portalId, srcPath, { qs });
-    logger.log('Fetched "%s" from portal %d successfully', src, portalId);
+    logger.log(
+      'Fetched "%s" from portal %d from the Desing Manager successfully',
```

Comment:
@williamspiro this is still inconsistent because the actual functionality differs a bit here. I believe DM assets are downloaded in bulk and then written. FM currently performs download/write in series.

Reply:
I think we can follow up in a separate PR and switch to the bulk model for file manager with a concurrency of 10.